### DISCUSSION TITLE
[Feature Request] Filter combinations

### DISCUSSION THREAD

**Discussion by superm1991** [ChristopherKlay/StadiaEnhanced#189] [Ideas & Requests]
Hi,

If you have the time to implement this, it would be cool to allow any complex combination of filters, such as:
- "Games OR Bundles" (equivalent to "NOT Add-ons")
- "NOT Wishlist"
- "Games AND Wishlist"
- etc...

**Comment by ChristopherKlay** [comment]
Will be added to the list (:
Thanks for the feedback!
